ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentFlows.tsx

```diff
@@ -0,0 +1,252 @@
+                isAnomalous: true,
+            },
+        ],
+    },
+    {
+        type: 'Deployment',
+        entity: 'Deployment 1',
+        namespace: 'naples',
+        direction: 'Ingress',
+        port: 'Many',
+        protocol: 'TCP',
+        isAnomalous: true,
+        children: [],
+    },
+    {
+        type: 'Deployment',
+        entity: 'Deployment 2',
+        namespace: 'naples',
+        direction: 'Ingress',
+        port: 'Many',
+        protocol: 'UDP',
+        isAnomalous: false,
+        children: [],
+    },
+    {
+        type: 'Deployment',
+        entity: 'Deployment 3',
+        namespace: 'naples',
+        direction: 'Egress',
+        port: '7777',
+        protocol: 'UDP',
+        isAnomalous: false,
+        children: [],
+    },
+];
+
+function DeploymentFlow() {
+    const initialExpandedRows = flows
+        .filter((row) => !!row.children.length)
+        .map((row) => row.entity); // Default to all expanded
+    const [expandedRows, setExpandedRows] = React.useState<string[]>(initialExpandedRows);
+
+    const setRowExpanded = (row: Flow, isExpanding = true) =>
+        setExpandedRows((prevExpanded) => {
+            const otherExpandedRows = prevExpanded.filter((r) => r !== row.entity);
+            return isExpanding ? [...otherExpandedRows, row.entity] : otherExpandedRows;
+        });
+
+    const isRowExpanded = (row: Flow) => expandedRows.includes(row.entity);
+
+    const totalFlows = flows.reduce((acc, curr) => {
+        // if there are no children then it counts as 1 flow
+        return acc + (curr.children.length ? curr.children.length : 1);
+    }, 0);
+
+    return (
+        <div className="pf-u-h-100 pf-u-p-md">
+            <Stack hasGutter>
+                <StackItem>
+                    <Toolbar>
+                        <ToolbarContent>
+                            <ToolbarItem>
+                                <TextContent>
+                                    <Text component={TextVariants.h3}>
+                                        {totalFlows} active flows
+                                    </Text>
+                                </TextContent>
+                            </ToolbarItem>
+                            <ToolbarItem alignment={{ default: 'alignRight' }} />
+                        </ToolbarContent>
+                    </Toolbar>
+                </StackItem>
+                <StackItem>
+                    <TableComposable aria-label="Deployment flow" variant="compact">
+                        <Thead>
+                            <Tr>
+                                <Th width={10} />
+                                <Th width={40}>{columnNames.entity}</Th>
+                                <Th width={20}>{columnNames.direction}</Th>
+                                <Th width={30}>{columnNames.portAndProtocol}</Th>
+                            </Tr>
+                        </Thead>
+                        {flows.map((row, rowIndex) => {
```

Comment:
what happens when there is an empty state? is an empty state possible here?

Reply:
I guess it'll just be empty. I'll have to add an empty state. @alanonthegit what should the empty state for the flows and baselines tab look like?